lusidtools/cocoon/cocoon.py

```diff
@@ -287,6 +287,47 @@ def load_instrument_property_batch(
 
         return results
 
+    @staticmethod
+    @run_in_executor
+    def load_portfolio_group_batch(
+            api_factory: lusid.utilities.ApiClientFactory, portfolio_group_batch: list, **kwargs
+    ) -> lusid.models.PortfolioGroup:
+        """
+        Upserts a batch of portfolios to LUSID
+        :param lusid.utilities.ApiClientFactory api_factory: The api factory to use
+        :param list[lusid.models.CreateTransactionPortfolioRequest] portfolio_batch: The batch of portfolios to create
+        :param kwargs: 'scope', 'code' arguments required for the API call
+        :return: lusid.models.PortfolioGroup: The response from LUSID
+        """
+
+        updated_request = group_request_into_one("CreatePortfolioGroupRequest", portfolio_group_batch, ["values"])
```

Comment:
portfolio_group_batch[0].__class__.__name__ will get you "CreatePortfolioGroupRequest" and will also work if the name of this class ever changes.

Reply:
Thanks